src/App.js

```diff
@@ -5,13 +5,13 @@ import Header from './components/Header/Header';
 class App extends Component {
     render() {
         return (
-          <body>
+          <div>
             <Header/>
             <div className="App" style={{ textAlign: 'center' }}>
               <h1>Hello World</h1>
```

Comment:
Instead of 'Hello World' you can write 'Kranti'.
In the next line, the catchphrase 'Join the Revolution' (as is there in the current website of the NGO) can be used.